tests/test_builder.py

```diff
@@ -111,7 +112,11 @@ def build():
 
     assert reporter.get_recorded_dependencies() == [
         'Website.lektorproject',
+        'content/blog/post1',
         'content/blog/post1/contents.lr',
+        'content/blog/post1/hello.txt',
+        'content/blog/post1/hello.txt.lr',
+        'models/blog-post.ini',
```

Comment:
Those seems to be added because of this line (but don't ask me why) https://github.com/lektor/lektor/pull/234/files#diff-dcf167e262df8b2c0acb0ad1fe5920c1R30